src/styles/components/Product.styles.js

```diff
@@ -0,0 +1,61 @@
+import styled from "styled-components";
+
+export const CardProduct = styled.div`
+  padding: 32px;
+  margin: 30px;
+  box-shadow: 0 0 20px rgba(0, 0, 0, 0.05), 0 0px 40px rgba(0, 0, 0, 0.08);
+  border-radius: 16px;
+  width: 250px;
+  transition: all 300ms ease;
+  &:hover {
+    transform: translateY(-5px);
+  }
+`;
+
+export const InfoProduct = styled.div`
+  display: flex;
+  align-items: center;
+  justify-content: space-between;
+`;
+
+export const NameProduct = styled.h1`
+  font-size: 22px;
+  font-family: Arial, Helvetica, sans-serif;
+`;
+
+export const PriceProduct = styled.p`
+  font-size: 21px;
+  color: #343a40;
+`;
+
+export const ImagenProduct = styled.img`
```

Comment:
seems like a mix of spanish and english, maybe it was `ImageProduct`?

Reply:
Yes jeje that was the idea